### ISSUE TITLE
`tpf.get_model()` does not work when FLUX_BKG is NaN

### ISSUE THREAD

**Issue by barentsen** [KeplerGO/lightkurve#200]
The solution will be to change this line:

```
flux_prior = GaussianPrior(mean=np.nanmean(self.flux_bkg),
                                       var=np.nanstd(self.flux_bkg)**2)
```

...it needs to be robust in case self.flux_bkg is nan.

**Comment by Emmavt** [comment]
@barentsen  Currently, tpf.get_model() does work when flux_bkg is NaN, it just sets the Background Prior with this NaN value for mean and var. Do we want to set it to a default value, or just not set the bkg prior if flux_bkg is NaN?

**Comment by barentsen** [comment]
@Emmavt I think we'll need something like this:

```Python
if np.all(np.isnan(self.flux_bkg)):
    # estimate background prior from self.flux instead of self.flux_bkg
```

**Comment by barentsen** [comment]
Fixed by #206 -- closing!
